tests/baselines/reference/tsc/incremental/incremental-declaration-doesnt-change/when-passing-filename-for-buildinfo-on-commandline.js

```diff
@@ -1,6 +0,0 @@
-//// [/lib/incremental-declaration-doesnt-changeOutput.txt]
-/lib/tsc --incremental --p src/project --tsBuildInfoFile src/project/.tsbuildinfo
```

Comment:
why does this file go away?

Reply:
oops this seems to have picked up #34487

Reply:
This file is moved to no-change folder instead